wallet/wallet.c

```diff
@@ -1133,7 +1134,7 @@ wallet_stmt2inflight(struct wallet *w, struct db_stmt *stmt,
 	if (!db_col_is_null(stmt, "lease_commit_sig")) {
 		lease_commit_sig = tal(tmpctx, secp256k1_ecdsa_signature);
 		db_col_signature(stmt, "lease_commit_sig", lease_commit_sig);
-		lease_chan_max_msat = db_col_int(stmt, "lease_chan_max_msat");
+		lease_chan_max_msat = db_col_u64(stmt, "lease_chan_max_msat");
```

Comment:
ACK, I was wondering how this works on the sqlite site but it looks like sqlite integers are 8 bytes anyways...